dashboard/app/controllers/levels_controller.rb

```diff
@@ -591,17 +591,28 @@ def extra_links
         path: script_level_path
       }
     end
+
+    parent_level_path_links = []
+    @level.levels_parent_levels&.each do |levels_parent_level|
+      parent_level_path_links << {
+        level_name: levels_parent_level.parent_level.name,
+        path: level_url(levels_parent_level.parent_level),
+        kind: levels_parent_level.kind,
+        position: levels_parent_level.position
+      }
+    end
     # TODO: Not present here, but present in original extra links. Some of these can be handled on the client side.
     # Anything project-specific should be handled via a separate API, as this controller has no context for projects.
-    # Gamelab show animation json, list contained levels, Blockly start/toolbox/etc, Blockly helpers, list of parent
-    # levels, all project validator links (should be handled elsewhere), abuse handlers (should be handled elsewhere).
+    # Gamelab show animation json, list contained levels, Blockly start/toolbox/etc, Blockly helpers, are not included yet
```

Comment:
Nit: would you mind removing Blockly start/toolbox/etc from list. Now included! (Mike H added toolbox recently.)
Blockly helpers are still not added.
![Screenshot 2025-01-30 at 11 33 23 AM](https://github.com/user-attachments/assets/46d35235-7690-4360-aa08-c99f81271333)

Reply:
done!